plugins/modules/system/java_cert.py

```diff
@@ -166,90 +170,193 @@
 '''
 
 import os
-import re
+import tempfile
+import random
+import string
 
 # import module snippets
 from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.six.moves.urllib.parse import urlparse
+from ansible.module_utils.six.moves.urllib.request import getproxies
+from re import search, sub
```

Comment:
Please se `re.search` and `re.sub` instead of importing the names directly. That makes the code a lot easier to read since it is clear these functions are from `re.`.

Reply:
done